bin/oscapd-cli

```diff
@@ -30,6 +30,75 @@ import sys
 from datetime import datetime
 import os.path
 
+class Task_accessor:
+
+    def __init__(self):
+        self._attributes = dict()
+
+    @staticmethod
+    def get_bool(values):
+        Task_accessor._expect_param_len(values, 1)
+        bool_val = Task_accessor._parse_bool(values[0])
+        return [bool_val,]
+
+    @staticmethod
+    def _parse_bool(val):
+        if val in ["1", "true", "yes"]:
```

Comment:
Please use `val.lower()` to also catch True, TRUE. We allow these variants elsewhere.